search/indexing_api.py

```diff
@@ -33,34 +33,49 @@
 SCRIPTING_LANG = 'painless'
 UPDATE_CONFLICT_SETTING = 'proceed'
 
-BASE_CONTENT_TYPE = {
+BASE_OBJECT_TYPE = {
     'object_type': {'type': 'keyword'},
     'author_id': {'type': 'keyword'},
     'author_name': {'type': 'keyword'},
+    'author_avatar_small': {'type': 'keyword'},
```

Comment:
@noisecapella I've added `author_avatar_small` to post and comment docs (derived from `profile_image` contained in their respective base serializers); profile docs will also include this and `author_avatar_medium` as well - does that sound good to you?